src/cheri/app-cap-description.adoc

```diff
@@ -1,16 +1,21 @@
 [appendix]
-[#app_cap_description]
-== Anatomy of Capabilities in {cheri_base_ext_name}
+[#app_cap_description, reftext="Default capability encoding"]
+== Default Capability Encoding in {cheri_base_ext_name}
+
+This chapter describes the default capability encoding for both {cheri_base64_ext_name} and {cheri_base32_ext_name}.
+It covers the following configurations:
+
+* <<uycfg>>.base=0, and
+* <<uycfg>>.features=0 (no levels support) or 1 (<<section_zylevels1>> support)
+
+If {cheri_default_ext_name} is supported, the specification includes encoding the <<cheri_execution_mode>> in the capability format.
```

Comment:
I've usually tried to say "supported by the platform" or something, to make it clear who's doing the relevant support.  Is there standard verbiage for this kind of composition in RISC-V specs?

Reply:
https://riscv-specs.timhutt.co.uk/spec/riscv-isa-release-4931695-2025-07-16/riscv-unprivileged.html#_risc_v_hardware_platform_terminology

Standard terminology is to say "if extension XYZ is implemented", looking through the unpriv spec